cli/flags/flags.go

```diff
@@ -147,8 +148,15 @@ func OperatorIDFlag(c *cobra.Command) {
 
 // ProofsFilePath add file path to proofs flag to the command
 func ProofsFilePath(c *cobra.Command) {
-	AddPersistentStringFlag(c, proofsFilePath, "proofs.json", "Path to proofs file", false)
+	AddPersistentStringFlag(c, proofsFilePath, "", "Path to proofs file, provide this OR a stringified proofs", false)
 }
+
+// ProofsStringFlag add proofs string flag to the command
+func ProofsStringFlag(c *cobra.Command) {
+	AddPersistentStringFlag(c, proofsString, "", "Stringified proofs, provide this OR a path to proofs file", false)
```

Comment:
how does it know that you gave the proof file if string is empty?
when does the error show up

Reply:
Here in `cli/utils/utils.go`:
```go
ProofsFilePath = viper.GetString("proofsFilePath")
ProofsString = viper.GetString("proofsString")
if ProofsFilePath == "" && ProofsString == "" {
    return fmt.Errorf("😥 Failed to get proofs from proofs string or path to proofs flag value")
}
if ProofsFilePath != "" && ProofsString != "" {
    return fmt.Errorf("😥 proofs can be provided either as a string, or path to a file, not both")
}
```